webapp/channels/src/workers/sw.js

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// Update this every time code is changed.
+// This is needed to delete old cache entries.
+const cacheName = 'v1';
+const HTML_REQUEST = new Request('root.html');
```

Comment:
Nit: Can we change this constant name to `ROOT_HTML_REQUEST`? I found it confusing at first, I thought it was something else.